README.md

```diff
@@ -89,7 +89,8 @@ So don't copy and paste the defaults from below!
 The _installer&#8209;config.txt_ is read in at the beginning of the installation process, shortly followed by the file pointed to with `online_config`, if specified.
 There is also another configuration file you can provide, _post&#8209;install.txt_, and you place that in the same directory as _installer&#8209;config.txt_.
 The _post&#8209;install.txt_ is executed at the very end of the installation process and you can use it to tweak and finalize your automatic installation.  
-The configuration files are read in as  shell scripts, so you can abuse that fact if you so want to. 
+The configuration files are read in as  shell scripts, so you can abuse that fact if you so want to.  
+If you want to write to the system that is being installed you will find it mounted under `/rootfs`  
```

Comment:
you might as well add that you can run commands in the installed system by doing ```chroot /rootfs cmd params```
and you also have ```/bootfs``` available...

:thought_balloon: on the other hand, I'd still refer anyone, who want's to take advantage of this, to look at [scripts/etc/init.d/rcS](https://github.com/debian-pi/raspbian-ua-netinst/blob/master/scripts/etc/init.d/rcS)